src/rsdp.rs

```diff
@@ -77,11 +85,16 @@ impl RsdpV2Tag {
         str::from_utf8(&self.signature).ok()
     }
 
-    /// The value to add to all the other bytes (of the Version 1.0 table) to calculate the Checksum of the table.
-    ///
-    /// If this value added to all the others and casted to byte isn't equal to 0, the table must be ignored. 
-    pub fn checksum(&self) -> u8 {
-        self.checksum
+    /// Validation of the RSDPv2 extended checksum
+    pub fn checksum_is_valid(&self) -> bool {
```

Comment:
Only added a validation of the RSDPv2 _extended_ checksum for the RsdpV2Tag. Should we have a RSDPv1 checksum validation and extended checksum validation function for the RsdpV2Tag?

Reply:
Yeah, the checksum of the ACPI 1.0 fields is still valid in the a RSDPv2 structure, but I don't know of a reason for a user to only check this. If we do find a reason to include it, it shouldn't be hard to add.